examples/source/multimedia-animations/Animated_hamburger_menu.html

```diff
@@ -0,0 +1,129 @@
+<!---
+ 
+teaserImage: '/static/samples/img/teaser/animated-hamburger-menu.jpg'
+author: CrystalOnScript
+ 
+--->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Animated hamburger menu</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- 
+    ## Introduction 
+    Small interactive UI animations communicate state and purpose to users. A common case is a hamburger menu button that animates into an "X" when open. This animation communicates that the user can press the same button again to close the menu. 
+  -->
```

Comment:
The ` <link rel="canonical"` below this comment will still be shown as a code block in the documentation.
That does not really make sense, does it?
Simply move the text at the top of the `CSS animations` comment block and separate them with a new line so that no code block will be shown between them.